resources/RELEASE_2022_04_03.md

```diff
@@ -0,0 +1,16 @@
+Quick notice for next week's update.
+_ _
+Our [plans on Skyra v7](https://skyra.notion.site/Skyra-v7-922ba06004654142a7b63347a92513a8) have changed. Discord's latest changes (and actions) threaten music bots as a whole as they must comply with third-party Terms of Service, which includes YouTube's, and because YouTube playback requires web scrapping and some reverse engineering (all of which is handled by an external service), we are forced to either:
+_ _
+- Make Aelia an Indie/Royalty Free radio.
+- Shut down Aelia.
+_ _
+However, the former comes with a lot of issues, and is no better than playing a 24/7 live from YouTube (e.g. [Lofi Girl](https://www.youtube.com/watch?v=5qap5aO4i9A)) or Twitch (e.g. [Monstercat](https://www.twitch.tv/monstercat)). After all, what makes music bots valuable is the fact that they are able to play anything the users want, e.g. to share a song somebody loves to their friends. And as such, __we will go for the latter__.
+_ _
+**What are the replacements?** At the moment, Spotify Premium allows you to play a lot of songs without ads, and has about as much control as most of the current music bots. In the future, we may see "Watch Together" in Discord, which is in essence, YouTube within Discord, and goes one step further by allowing videos to be shared.
+_ _
+**What about Skyra?** We will __remove the music module__, all of it. Discord has started unverifying bots that breaks the aforementioned ToS without a chance to appeal, and this is a measure we cannot risk getting into.
+_ _
+We are sorry we have to take this kind of measure with such short notice, but having Skyra up and running for everyone is our top 1 priority.
```

Comment:
```suggestion
We are sorry we have to take this kind of measure with such short notice, but having Skyra up and running for everyone is our number 1 priority.
```

`top 1 priority` is a [lexical blend](https://en.wikipedia.org/wiki/Blend_word). it's either `top priority` or `number 1 priority`.